modules/update-restricted.adoc

```diff
@@ -20,6 +20,7 @@ If you have a local OpenShift Update Service, you can update by using the connec
 * You applied the release image signature ConfigMap for the new release to your cluster.
 * You obtained the sha256 sum value for the release from the image signature ConfigMap.
 * Install the OpenShift CLI (`oc`), version 4.4.8 or later.
+//Do we still need to specify this, since 4.4 is out of support?
```

Comment:
@codyhoag, PTAL?

Reply:
@kalexand-rh this prereq can be removed :+1: